modules/pam_faildelay/pam_faildelay.c

```diff
@@ -79,13 +79,13 @@
 
 /* --- authentication management functions (only) --- */
 
-static long parse_delay(const char *val)
+static long long parse_delay(const char *val)
 {
-    long delay;
+    long long delay;
     char *endptr;
 
-    delay = strtol (val, &endptr, 10);
-    if (delay == 0 && val == endptr)
+    delay = strtoll (val, &endptr, 10);
+    if (delay < 1 || val == endptr || *endptr != '\0')
       return -1;
```

Comment:
This looks like an unintended change of behavior: "0" which used to be a valid delay is now turned into an error.

Reply:
Oh, true. Fixed.